prime-router/src/test/kotlin/credentials/CredentialTests.kt

```diff
@@ -2,10 +2,23 @@ package gov.cdc.prime.router.credentials
 
 import assertk.assertThat
 import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
 import assertk.assertions.isTrue
 import kotlin.test.Test
 
 internal class CredentialTests {
+    private val ppkCredentialWithPass = """
```

Comment:
Dumb suggestion, but make the values semi-constants - maybe something that is generated at run time (Uuid?) so that they change at every run - that you then re-reference in the test to check 'was this value picked up'.
values such as '`key`' or '`keyPass`' or '`pass`' are a bit too generic.

Reply:
yea, that makes sense